src/cyc_limits.h

```diff
@@ -5,14 +5,19 @@
 #include <limits>
 
 namespace cyclus {
+
+/// generic epsilon values
+extern double cy_eps;
```

Comment:
I'm a little confused about where these have their primary definition (and why).  Should they be defined primarily here in `cyc_limits`?  Not sure if there is a single right answer?